modules/apps/frontend-js/frontend-js-web/test/liferay/util/format_storage.es.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+import formatStorage from '../../../src/main/resources/META-INF/resources/liferay/util/format_storage.es';
+
+describe('Liferay.Util.formatStorage', () => {
+	it('should return formatted string with size and suffix type KB if size input type is number <= 1048575 and options is default value', () => {
+		expect(formatStorage(10400)).toEqual('10KB');
+	});
+
+	it('should return formatted string with negative size and suffix type KB if size input type is negative number and options is default value', () => {
+		expect(formatStorage(-10400)).toEqual('-10KB');
+	});
+
+	it('should return formatted string with zero and suffix type KB if size input type is 0 and options is default value', () => {
+		expect(formatStorage(0)).toEqual('0KB');
+	});
+
+	it('should return formatted string with size and suffix type MB if size input type is number >= 1048576 and options is default value', () => {
+		expect(formatStorage(1048576)).toEqual('1.0MB');
+	});
+
+	it('should return NaN with suffix type KB if size input type is String and options is default value', () => {
+		expect(formatStorage('Test')).toEqual('NaNKB');
```

Comment:
This behavior is not right. If the parameter is not of the right type, we should throw an error. 

See for example https://github.com/liferay/liferay-portal/blob/c4305106ea60aafe4283eb18954a0fe7a141a328/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/format_xml.es.js#L58 
and https://github.com/liferay/liferay-portal/blob/c4305106ea60aafe4283eb18954a0fe7a141a328/modules/apps/frontend-js/frontend-js-web/test/liferay/util/format_xml.es.js#L7